Source/WebCore/loader/archive/mhtml/MHTMLArchive.cpp

```diff
@@ -57,8 +57,8 @@
 
 namespace WebCore {
 
-const char* const quotedPrintable = "quoted-printable";
-const char* const base64 = "base64";
+static constexpr auto quotedPrintable = "quoted-printable"_s;
```

Comment:
Pretty sure we don’t need `static` if we are using `constexpr` at the namespace level like this.